### PR TITLE
Env utilisation

### PR DESCRIPTION
**Pull request by delcroip**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,4 +10,17 @@ RUN dotnet publish OpenImis.RestApi/OpenImis.RestApi.csproj -c $BUILD-FLAVOUR -o
 FROM mcr.microsoft.com/dotnet/core/aspnet:2.1
 WORKDIR /app
 COPY --from=build-env /app/OpenImis.RestApi/out .
-ENTRYPOINT ["dotnet", "OpenImis.RestApi.dll"]
+
+ENV DB_HOST=Server
```
Comment (dragos-dobre) [inline Dockerfile:14]: Should this be taken form .env file?

File: docker-compose.yml [hunk 2]
```diff
@@ -5,10 +5,17 @@ services:
       context: ./
       args:
         BUILD-FLAVOUR: ${BUILD-FLAVOUR:-Release}
+    environements:
+      - DB_HOST=Server
```
Comment (dragos-dobre) [inline docker-compose.yml:9]: Should this be taken from .env file?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (delcroip) [reply docker-compose.yml:9]: not really the goal is not to use this compose file, it is more an example

File: Dockerfile (hunk 1 above)
Comment (delcroip) [reply Dockerfile:14]: No, here I just set the default value, we can remove the value or even the line but I think it is good for documentation purpose

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**